packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

```diff
@@ -528,6 +534,7 @@ The Authenticator object defines the following properties:
 | `lastUpdated` | String (ISO-8601)                                               | Timestamp when the Authenticator was last modified.                             |
 | `name`        | String                                                          | Display name of this Authenticator.                                             |
 | `type`        | String (Enum)                                                   | The type of Authenticator. Possible values: `password`, `security_question`, `phone`, `email`, `security_key`                            |
+| `allowedFor`        | String (Enum)                                                   | The allowed types of usages for the Authenticator. Possible values: `recovery`, `sso`, `any`, `none`                            |
```

Comment:
One suggestion: as per my Slack question, if allowed types are limited, include the serial "and" and remove "possible"; that is, "Values include `recovery`, `sso`, `any`, and `none`"
(Same comment for the `type` property above `allowedFor`)

Reply:
Done!